### ISSUE TITLE
problem with multiple bots

### ISSUE THREAD

**Issue by robingroppe** [MusicGenerator/mumble-ruby-pluginbot#218]
I had some trouble with a setup of three bots.
Only the second and third one were working properly.

As it turns out the logfile is hardcoded in the `mumble-ruby-pluginbot/config/config.yml` and never overwritten in the `src/botX_conf.yml`.

It seems that it causes strange things when all bots try to write in the bot1.log ;).

Please add `logfile: "/home/botmaster/logs/bot1.log"` to the `mumble-ruby-pluginbot/templates/override_config.yml`

**Comment by Natenom** [comment]
The reason we created the override_config was to provide a simple config file with the most important configuration options only at a time when the configuration file was really huge. It was not meant that the admin does not need to look into the default config file anymore.

Now the configuration file is much smaller and also split in to a main file and plugin specific files. I just compared config/config.yml with templates/override_config.yml and they differ in a few lines only.

However it makes sense to add `logfile: "/home/botmaster/logs/bot1.log"` to the override_config.yml, tank you :)

I don't consider this as a bug but this entry will be there in 0.10.2. Until then one needs to add it himself.

@dafoxia: Do we still want that override config file? Note that we also override some plugin settings there.
